flapp-webapp/src/App.vue

```diff
@@ -1,20 +1,23 @@
 <template>
   <div class="app-outer fill-body" id="app">
     <Navbar />
-    <main class="app-content">
-      <SurveySidebarComponent />
-      <SurveyComponent />
+    <main class="app-content fill-body">
+      <SurveySidebarComponent
+        v-bind:selectedForm="selectedForm"
+        @updated-selection="onSidebarClicked"
+      />
+      <SelectedForms v-bind:form="selectedForm" />
```

Comment:
Please try to use the short forms consistently, `:var="x"` is equivalent to `v-bind:var="x"` and `@var="x"` is equivalent to `v-on:var="x"`